doc/getting-started/installation.md

```diff
@@ -34,8 +38,9 @@ curl "https://github.com/jolicode/castor/releases/latest/download/castor.linux-a
     (echo "Could not install castor. Is the target directory writeable?" && (exit 1))
 ```
 
-> **Note:** Castor is also released for Linux running on ARM64 architectures.
-> In such case, replace `linux-amd64` with `linux-arm64` in the URL.
+> [!NOTE]
```

Comment:
I started to make a sub chapter like it's done for mac, but anyway, let's keep it like that for now